src/Loading/babylon.sceneLoader.ts

```diff
@@ -1,16 +1,27 @@
 ﻿module BABYLON {
+    export interface ISceneLoaderPluginExtensions {
+        [extension: string]: {

```

Comment:
Can you please explain me what are doing the lines 3, 4 and 5? I really can't figure this out! :)